src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -232,6 +232,8 @@ def update_search_space(self, search_space):
         search_space : dict
         """
         self.json = search_space
+        randint_to_quniform(self.json)
```

Comment:
I checked smac, hyperband, and bohb, the same modification would work. Could add this line in them?

Reply:
I think that owner add them would be better.